setup_ros_catkin_make_workspace.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# EECS 376/476 Mobile Robotics
+# Workspace setup
+# Luc Bettaieb & Russell Jackson
+# v 0.25
+
+echo "Setting up workspace."
+
+USERNAME=$1
+EMAIL=$2
+
+if [ "$USERNAME" != "" ] || [ "$EMAIL" != "" ];
+then
+  source ~/.bashrc
+  echo -e "\e[34m >>> Sourced ~/.bashrc to obtain the ros distro path. \e[39m"
+
+  mkdir -p ~/ros_ws/src
+	
+  cd ~/ros_ws/src  && catkin_init_workspace
+  cd ~/ros_ws && catkin_make
+
+  mkdir -p ~/ros_ws/src/cwru
+  mkdir -p ~/ros_ws/src/local_code
+
+  git config --global user.name "$USERNAME"
+  git config --global user.email "$EMAIL"
+
+  cd ~/ros_ws/src/cwru && git clone https://github.com/cwru-robotics/cwru_msgs.git
```

Comment:
might wanna move this file into somewhere more specific and have it do more specific things